src/sentry/monitors/clock_tasks/check_missed.py

```diff
@@ -158,6 +159,9 @@ def mark_environment_missing(monitor_environment_id: int, ts: datetime):
         monitor.schedule,
     )
 
+    update_monitor_environment(
```

Comment:
Let's leave a comment here about using the existing `last_checkin` since we don't want to move it forward and confuse the user, since these check-ins are synthetic